commands/sync.go

```diff
@@ -41,9 +49,24 @@ var SyncStatusCmd = &cli.Command{
 			return err
 		}
 
-		fmt.Println("sync status:")
+		short := cctx.Bool("short")
```

Comment:
Wouldn't it be cleaner to add an `output` flag which can be `text` or `json`. The default is text, but you could add json output and then query that using `jq` or similar?